packages/frontend/src/constants.ts

```diff
@@ -102,26 +103,29 @@ export const CARROT_DOMAIN = prod
     : "staging.carrot.community";
 
 export interface NavbarLink {
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    Component?: FunctionComponent<any>;
     title: string;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    Icon: FunctionComponent<any>;
     to: string;
     external: boolean;
+    highlighted?: boolean;
 }
 
 export const NAVBAR_LINKS: NavbarLink[] = [
     {
-        Component: NavLink,
-        title: "About",
-        to: `https://www.${CARROT_DOMAIN}`,
-        external: true,
-    },
-    {
-        Component: NavLink,
         title: "Campaigns",
+        Icon: Grid,
         to: "/campaigns",
         external: false,
     },
+    {
```

Comment:
We should probably comment this out for now at least until the new "create" page is ready design wise.

Reply:
Fixed in 34e358a7437afad6342fd9a69fb71f1c56d5c4bb.